configs/app/features/pools.ts

```diff
@@ -0,0 +1,28 @@
+import type { Feature } from './types';
+
+import { getEnvValue } from '../utils';
+
+const contractInfoApiHost = getEnvValue('NEXT_PUBLIC_CONTRACT_INFO_API_HOST');
+const dexPoolsEnabled = getEnvValue('NEXT_PUBLIC_DEX_POOLS_ENABLED');
+
+const title = 'Pools';
```

Comment:
```suggestion
const title = 'DEX pools';
```

Let's be consistent with the documentation.